lib/ACLMicroservicesBuilder.js

```diff
@@ -229,42 +226,39 @@ ACLMicroservicesBuilder.prototype._executeSavePermissions = function(args, callb
 
   if(args.ent.id) { // update
 
-    self.ACLMicroservicesBuilder._loadAndAuthorize(entityDef, args.ent.id, args.cmd, roles, context, args.showSoftDenied$, function(err, dbEntity) {
+    self.ACLMicroservicesBuilder._loadAndAuthorize(entityDef, args.ent.id, args.cmd, roles, context, args.showSoftDenied$,args.ent, function(err, dbEntity) {
 
       if(err) {
 
-        callback(err, undefined)
-
-      } else {
-
-        // also execute permission checks on the new attributes
-        self.ACLMicroservicesBuilder._deepAuthorize(entityDef, args.ent, args.cmd, 'save_new', roles, context, true, args.showSoftDenied$, function(err, filteredEntity) {
-
-          if(err) {
+        return callback(err, undefined)
 
-            callback(err, undefined)
+      }
 
-          } else {
+      // also execute permission checks on the new attributes
+      self.ACLMicroservicesBuilder._deepAuthorize(entityDef, args.ent, args.cmd, 'save_existing', roles, context, true, args.showSoftDenied$, function(err, filteredEntity) {
```

Comment:
it looks that a ``save_new`` changed to a ``save_existing``. It changes the semantics of these 2 actions.